lib/spack/spack/test/cmd/install.py

```diff
@@ -142,7 +142,8 @@ def test_install_output_on_build_error(mock_packages, mock_archive, mock_fetch,
                                        config, install_mockery, capfd):
     # capfd interferes with Spack's capturing
     with capfd.disabled():
-        out = install('build-error', fail_on_error=False)
+        out = install('-v', 'build-error', fail_on_error=False)
+
```

Comment:
Did you mean to leave this in?

Reply:
Yes, I think @tgamblin mentioned in slack that if we add the `-v` option the test works (but I haven't seen it run yet).